### DISCUSSION TITLE
Making transpose function as a property

### DISCUSSION THREAD

**Discussion by ZAARAOUI999** [adtzlr/tensortrax#96] [General]
Hi Mr Andreas, I wanna share with you this small trick, it looks like not interesting but...
So, my idea is just to add ```@property``` before this function in the ```Tensor``` class:
```python
@property
def T(self):
        return transpose(self)
```
So the user can get the transpose with just typing ```Tensor.T``` rather than ```Tensor.T()```
Thanks.

**Comment by adtzlr** [comment]
Hi @ZAARAOUI999!

Thank you for your valuable idea - I didn't know that before.

**Reply by adtzlr** [reply]
I created an issue based on this discussion, see #97
